doc/integrator/checker.rst

```diff
@@ -50,6 +50,24 @@ The checker use the following configuration structure:
         checker:
             forward_headers: ['Cookie', 'Authorisation']
 
+.. note::
+
+    The checker assumes that it can access the c2cgeoportal services via ``http://localhost``.
+    If this is not allowed on your server, you can override this behaviour as follows.
+    In the ``vars`` - ``checker`` configuration (see above), add the following:
+
+    .. code:: yaml
+
+            rewrite_as_http_localhost: False
+
+    Now, in your project YAML configuration, instead of defining the ``checker_path``,
```

Comment:
With file is it, it seem to be the same as the previous one...

Reply:
no, it is the one that currently has "checker_path". So this would be replaced by "checker_url".